### PR TITLE
Added contact form

### PR DESCRIPTION
**Pull request by Amrhub**
@Amrhub and @AlexRS90  worked as "Pair programming" to add contact form using <a href="https://formspree.io/html">Formspree</a> and give it proper styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,14 +177,35 @@ <h3 class="Project-title">
...
+
+        <footer class="footer">
+            <section class="Contact-us">
+                    <h1 class="Section-title">
+                        CONTACT US
+                    </h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                    </p>
+                    <form class="Contact-us__Form" id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="#" method="post">
```
Comment (carlos-ssh) [inline index.html:196]: -After submit the form i have this error message:
Screen Shot 2021-09-21 at 16 18 18

- Remember that your `` action `` value should be similar to `` https://formspree.io/f/{form_id} `` .

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-> Portfolio: mobile version
+> Portfolio: mobile version - Contact form
 
 ![screenshot](./app_screenshot.png)
 
-This project is a mobile version for About-Me & Works sections of the portfolio.
+This project is a mobile version added contat form
 
 ## Built With
```
Comment (carlos-ssh) [inline README.md:11]: Provide steps on how to install your project locally.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Amrhub , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Amrhub) [reply README.md:11]: It's already provided

File: index.html (hunk 1 above)
Comment (Amrhub) [reply index.html]: I removed it because it does a linter error and added so, you can see it  says "{}" is not supported in HTML checker

**Review by ZeenatLawal — APPROVED** [review]
Hi @Amrhub and @AlexRS90, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**